docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+
+As a [best practice](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#lock-away-credentials), do not use the AWS account root user for any task where it's not required. Instead, create a new IAM user for each person that requires administrator access. Then make those users administrators by placing the users into an "Administrators" user group to which you attach the `AdministratorAccess` managed policy.
+
+If its your first time creating a new IAM user, please refer to [Creating your first IAM admin user and user group](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html) for more information. Otherwise, see [Creating an IAM user in your AWS account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) for more information on how to create a IAM user.
+
+Follow these steps to set up your access keys and user accounts:
+
+- Go to [IAM in your AWS Console](https://console.aws.amazon.com/iam/home), then **Users**, and **Add Users**;
+- Give the user a name, and tick **Access Key - Programmatic access**
+- Click **Next**
+
+![alt text for screen readers](/img/how-tos-aws-new-iam-user.png "Text to show on mouseover")
+
+- Select **Attach existing policies directly**, then select `AdministratorAccess` from the list of policies. For more information, please refer to [Policies and permissions in IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html);
+- Then proceed with the new user creation setup.
+
+Upon generation, the IAM role will provide a public **Access Key ID** and **Secret Access Key**. Download this file for reference later.
+
+:::warning Warning
+The **Secret Access Key** provides access to your AWS user account. It should be treated like any other secret credentials. Specifically, it should never be checked into source control.
+:::
+
+By default, Nebari will try to use the credentials associated with the current AWS infrastructure/environment for authentication. Please keep in mind that Nebari will only use these credentials to create the first roles and stricter permissions for Nebari's internal components. Refer to [Conceptual guides] for more information on how Nebari's components are secured.
+
+Provide authentication credentials to Nebari by setting the following environment variables:
+
+```bash
+export AWS_ACCESS_KEY_ID="Access Key ID"
+export AWS_SECRET_ACCESS_KEY="Secret Access Key"
+```
+
+If you are using an already existing IAM user or want to use a different par of access keys, please refer to [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey) for detailed information on how to manage your IAM user's access keys.
```

Comment:
I think this should be relocated to be the second paragraph in this "Authentication" section -- it should be somewhere that someone who already has an IAM user and keys will find it quickly, instead of having to read to the bottom of the section.

Is there any reason other than wanting to use an existing IAM user that would motivate someone to use a different pair of access keys? If not, then "If you are using an existing IAM user or want to use a different pair of access keys" is redundant.

```suggestion
If you are using an existing IAM user or want to use a different pair of access keys, please refer to [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey) for detailed information on how to manage your IAM user's access keys.
```